lib/prehog/proto/buf.yaml

```diff
@@ -1,6 +0,0 @@
-version: v1
-lint:
-  use:
-    - DEFAULT
-    - UNARY_RPC
```

Comment:
lib/prehog used to use [UNARY_RPC](https://docs.buf.build/lint/rules#unary_rpc). However, I don't think it's possible to enable a single category only for a certain path, unless I enable the category for all paths and then ignore it for all paths except the one that needs it.

When it comes to protos in `proto`, only lib/teleterm uses streaming RPCs. Should I enable UNARY_RPC globally and ignore lib/teleterm? We could add a comment like "If you're sure you want to use streaming RPCs, add protos to ignored paths".

Reply:
I like that idea, +1. We should do it for api/proto too.

You can leave my handle in a comment, so people can look to me for details.